src/web/CustomScrollbar.ts

```diff
@@ -511,14 +507,22 @@ export class Scrollbar {
             }
         }
         Scrollbar._installStyleSheet();
-        this._tryLtrOverride();
         this._addScrollbars();
         this.show();
-        this.update();
         this._container.addEventListener('mouseenter', this._updateCallback);
+
+        // Defer remaining init work to avoid triggering sync layout
+        this._asyncInitTimer = window.setTimeout(() => {
+            this._tryLtrOverride();
+            this.update();
+        }, 0);
```

Comment:
You should set this._asyncInitTimer to undefined within the callback. Otherwise clearInterval will be called on an already-fired timer in dispose.

Reply:
Fixed in 66bfc976b23. Thanks!